resources/sugarcrm/sugarjob.cpp

```diff
@@ -223,4 +223,9 @@ Sugarsoap *SugarJob::soap()
     return d->mSession->soap();
 }
 
+bool SugarJob::getTryRelogin() const
```

Comment:
The Qt coding style is "no get prefix". But of course tryRelogin() would sound like an action ("try now") rather than a getter. So I suggest this:
   bool SugarJob::shouldTryRelogin() const